src/languages/mathematica.js

```diff
@@ -6,61 +6,118 @@ Website: https://www.wolfram.com/mathematica/
 Category: scientific
 */
 
-export default function(hljs) {
+/*
+  The ./lib/mathematica.js file was created with Wolfram Mathematica 12.1.1 under Linux
+  with the following code:
+  $outputFile="/path/to/highlight.js/src/languages/lib/mathematica.js";
+  getStrippedContextNames[context_String]:=Block[
+    {
+      $ContextPath={context}
+    },
+    Union[Names@RegularExpression@StringJoin[context,"\\$?[A-Z]\\w*"]]
+  ];
+  $symbols=getStrippedContextNames["System`"];
+  Export[$outputFile,
+    With[
+      {
+        quoted="\""<>#<>"\""&/@$symbols
+      },
+      StringTemplate["export const SYSTEM_SYMBOLS = [\n`syms`\n];\n"][
+        <|"syms"->StringRiffle[Map["  " <> #&,quoted],",\n"]|>
+      ]
+    ],"String"]
+*/
+import * as Mathematica from './lib/mathematica.js';
+import * as regex from '../lib/regex.js';
+
+// @ts-ignore
+export default function(highlightJS) {
```

Comment:
Please revert this to match every other grammer, `hjls`, etc...

Reply:
OK. It wasn't a (camel-case) word and was flagged by my spell-check.

Reply:
> other grammer, hjls, etc

This is why I renamed it to something readable.. even you mistyped it :joy:

Reply:
Legacy, consistency, etc... and it's to match the fact that `hljs` is our global. :-)  Not something we're going to change at this time. :)